docs/howto/grape.md

```diff
@@ -0,0 +1,27 @@
+# Integration with Grape
+
+To add Grape support, `require 'grape/active_model_serializers'` in the base of your Grape endpoints like so:
+
+```ruby
+module Example
+  class Dummy < Grape::API
+    require 'grape/active_model_serializers'
+    mount Example::V1::Base
+  end
+end
+```
+
+Then add the formatter and helpers to the endpoints (a `Defaults` module is used here which would be included in all endpoints):
+
+```ruby
+module Example::V1::Defaults
+  extend ActiveSupport::Concern
+
+  included do
+    formatter :json, Grape::Formatters::ActiveModelSerializers
+    helpers Grape::Helpers::ActiveModelSerializers
+  end
+
+end
```

Comment:
we can make that mixin here, so that the instructions are `include  Grape::ActiveModelSerializers` or something like that. Thoughts?